packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1417,20 +1418,18 @@ export abstract class Chart extends Observable {
         debug(`Chart.applyAxes() - creating new axes instances; seriesStatus: ${seriesStatus}`);
         chart.axes = this.createAxis(axes, skip);
 
-        const axisGroups: { [Key in ChartAxisDirection]: { id: string; node: Node }[] } = {
-            [ChartAxisDirection.X]: [],
-            [ChartAxisDirection.Y]: [],
-        };
+        if (registerRegions) {
+            const axisGroups: { [Key in ChartAxisDirection]: { id: string; node: Node }[] } = {
+                [ChartAxisDirection.X]: [],
+                [ChartAxisDirection.Y]: [],
+            };
 
-        chart.axes.forEach((axis) => {
-            const { id } = axis;
-            const node = axis.getRegionNode();
-            if (node != null) {
-                axisGroups[axis.direction].push({ id, node });
+            for (const axis of chart.axes) {
+                if (axis instanceof CartesianAxis) {
```

Comment:
Nit: I'm not sure about this change, it's probably better if `Chart` doesn't vary it's behaviour here based upon the type of `axis`, but the `axis` controls this.

Reply:
My reasoning is that this is a patch and a patch shouldn't change public APIs because that's how a temporarily problem becomes permanent :)